packages/gatsby-source-strapi/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "gatsby-source-strapi",
+  "version": "3.0.0",
+  "description": "Gatsby source plugin for building websites using Strapi as a data source",
+  "keywords": [
+    "gatsby",
+    "gatsby-plugin",
+    "gatsby-source-plugin",
+    "strapi"
+  ],
+  "directories": {
+    "dist": "./dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gatsby-uc/plugins.git",
+    "directory": "packages/gatsby-source-strapi"
+  },
+  "bugs": {
+    "url": "https://github.com/gatsby-uc/plugins/issues"
+  },
+  "homepage": "https://github.com/gatsby-uc/plugins/packages/gatsby-source-strapi#readme",
+  "author": {
+    "email": "hi@strapi.io",
+    "name": "Strapi Solutions",
+    "url": "http://strapi.io"
+  },
+  "contributors": [
+    "Laurens Kling <laurens@goedideemeida.nl>"
+  ],
+  "license": "MIT",
+  "dependencies": {
+    "axios": "0.24.0",
+    "commonmark": "^0.30.0",
+    "gatsby-source-filesystem": "^5.2.0",
+    "lodash": "^4.17.21",
+    "qs": "^6.10.3"
+  },
+  "scripts": {
+    "build": "babel src --out-dir dist"
+  },
+  "devDependencies": {
+    "@babel/cli": "7.14.5",
+    "@babel/core": "7.14.6"
+  },
+  "peerDependencies": {
+    "gatsby": "^4.0.0 || ^5.0.0",
+    "gatsby-source-filesystem": "^4.5.0 || ^5.0.0"
```

Comment:
This doesn't go here if it's in dependencies.

Reply:
Ok sorry for the back and forth here. I realized I should check to see what the Gatsby team is doing. Checkout it does vary but  
```
 "peerDependencies": { 
     "gatsby": "^5.0.0", 
     "gatsby-plugin-image": "^3.0.0", 
     "gatsby-plugin-sharp": "^5.0.0", 
     "gatsby-transformer-sharp": "^5.0.0"
}
```

Is how they're doing it in source WordPress, and that all makes sense to me.

You could set them as a.tional peerdeps if you wanted...https://stackoverflow.com/questions/62047806/how-do-i-set-a-peer-dependency-optional/66228639#66228639

Reply:
Yeah I guess you'd want to pin a version used if the user ends up using images. I'll add it again. Supporting both Gatsby 4 and 5